member.php

```diff
@@ -783,7 +783,64 @@
 	{
 		$plugins->run_hooks("member_register_start");
 
-		$validator_extra = '';
+		// JS validator extra
+		if($mybb->settings['maxnamelength'] > 0 && $mybb->settings['minnamelength'] > 0)
+		{
+			$lang->js_validator_username_length = $lang->sprintf($lang->js_validator_username_length, $mybb->settings['minnamelength'], $mybb->settings['maxnamelength']);
+		}
+
+		$validator_javascript = "<script type=\"text/javascript\">
```

Comment:
We can certainly live with `1` for now and can always change it later if there is a sensible request to change it for a good reason.